common.sh

```diff
@@ -471,3 +471,5 @@ if [[ -n "$MIRROR_IMAGES" || -z "${IP_STACK:-}" || "$IP_STACK" = "v6" ]]; then
        export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="${LOCAL_REGISTRY_DNS_NAME}:${LOCAL_REGISTRY_PORT}/${LOCAL_IMAGE_URL_SUFFIX}:latest"
    fi
 fi
+
+export AGENT_E2E_TEST_TUI_BAD_DNS=${AGENT_E2E_TEST_TUI_BAD_DNS:-}
```

Comment:
I'm wondering if we could find a better name for this.  I see some conflict with `AGENT_E2E_TEST_SCENARIO` config var, which indicates a different setup (the topology to be deployed), while here we're really naming a specific e2e test. I do not exclude the possibility to have some additional tests in future, so maybe it could be worth spending some time to find a proper prefix and/or naming convention.

Reply:
Yes, the name can be confusing. How about AGENT_TEST_CASES?